Config.md

```diff
@@ -0,0 +1,48 @@
+# Using Configs
+
+nx-parallel algorithms have a `joblib.parallel_config` context manager which has the `joblib.Parallel` instance. You change the config parameters of the internal `joblib.parallel_config` context manager using networkx's context manager, like this:
+
+```
+>>> import networkx as nx
+>>> nx.config # default configs
+NetworkXConfig(backend_priority=[], backends=Config(parallel=ParallelConfig(backend=None, n_jobs=None, verbose=0, temp_folder=None, max_nbytes='1M', mmap_mode='r', prefer=None, require=None, inner_max_num_threads=None, backend_params={})), cache_converted_graphs=True)
+>>>
+>>> nxp_config = nx.config.backends.parallel
+>>> nxp_config.backend = "loky" # global config
+>>>
+>>> G = nx.complete_graph(20)
+>>> # using context manager
+>>> with nxp_config(n_jobs=4): # backend -> loky , n_jobs -> 4 , verbose -> 0
+...     nx.square_clustering(G, backend="parallel")
+...     nxp_config
+...     print()
+...     with nxp_config(n_jobs=3, verbose=15): # backend -> loky , n_jobs -> 3 , verbose -> 15
```

Comment:
Why are the context manager calls set within the previous context manager call? Shouldn't they be separate -- in the sense that we end one context manager before starting the next configuration setting -- that is, should we dedent these inner context manager calls?

(and can we put the comments on the previous line so they don't make the line so long?

Reply:
here, I wanted to show how context managers inherit from the outer(or previous) context manager. And how we can keep making configs more and more specific by "(sort-of)nesting" context managers. LMK what you think. I think we can add another context manager that is independent of all these.

Reply:
If that is what we are showing, then let's put some words saying that it is one of the points/features of these examples.